lib/galaxy_test/api/test_notifications.py

```diff
@@ -0,0 +1,50 @@
+from galaxy_test.base.populators import DatasetPopulator
+from ._framework import ApiTestCase
+
+
+class NotificationsApiTestCase(ApiTestCase):
+    def setUp(self):
+        super().setUp()
+        self.dataset_populator = DatasetPopulator(self.galaxy_interactor)
+
+    def test_index(self):
+        payload = {"message_text": "Test Message"}
+        notification_response = self._post("notifications", payload, json=True)
+        notification_response.raise_for_status()
+        payload = {"limit": 5, "offset": 0}
```

Comment:
The types of your notification dictionary are changing, that's why you're getting:
```
lib/galaxy_test/api/test_notifications.py:14: error: Dict entry 0 has
incompatible type "str": "int"; expected "str": "str"  [dict-item]
            payload = {"limit": 5, "offset": 0}
```